<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doktorekle</class>
 <widget class="QDialog" name="doktorekle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>178</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>396</width>
    <height>178</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>519</width>
    <height>178</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Yeni Doktor Ekle</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblAdi">
       <property name="text">
        <string>&amp;Adı</string>
       </property>
       <property name="buddy">
        <cstring>leAdi</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leAdi"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSoyadi">
       <property name="text">
        <string>&amp;Soyadı</string>
       </property>
       <property name="buddy">
        <cstring>leSoyadi</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leSoyadi"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblTel">
       <property name="text">
        <string>&amp;Telefon</string>
       </property>
       <property name="buddy">
        <cstring>leTel</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leTel"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDipno">
       <property name="text">
        <string>&amp;Diploma No</string>
       </property>
       <property name="buddy">
        <cstring>leDipno</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leDipno"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblUzmalani">
       <property name="text">
        <string>&amp;Uzmanlık Alanı</string>
       </property>
       <property name="buddy">
        <cstring>leUzmalani</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leUzmalani"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="ButtonKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonVazgec">
       <property name="text">
        <string>Vazgeç</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonKaydet</sender>
   <signal>clicked()</signal>
   <receiver>doktorekle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonVazgec</sender>
   <signal>clicked()</signal>
   <receiver>doktorekle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
